curriculum/challenges/english/01-responsive-web-design/css-piano/part-037.md

```diff
@@ -0,0 +1,103 @@
+---
+id: 612ea4c4993aba52ab4aa32e
+title: Part 37
+challengeType: 0
+dashedName: part-37
+---
+
+# --description--
+
+Now it is time to use the pseudo selectors you prepared for earlier. To create the black keys, add a new `.key.black--key::after` selector. This will target the elements with the class `key black--key`, and select the pseudo-element afer these elements in the HTML.
```

Comment:
```suggestion
Now it is time to use the pseudo-selectors you prepared for earlier. To create the black keys, add a new `.key.black--key::after` selector. This will target the elements with the class `key black--key`, and select the pseudo-element after these elements in the HTML.
```